README.md

```diff
@@ -94,27 +94,11 @@ cd snowflake-connector-nodejs
 npm install .
 ```
 
-3. Install the nyc module which displays the code coverage
+3. Show code coverage report
```

Comment:
1. maybe we can get rid of step 1 and 2 since in the test section we don't have info how to install code
2. Then we can get rid of numbering
3. In this heading let's add info that it's running test with gathering code coverage